db/sites-service/sitesList.js

```diff
@@ -66,17 +66,17 @@ module.exports = [
         mandatory: "true",
         symbols: [
           {
-            name: "Island Mode",
+            name: "Tower Mode",
             value: "1",
             label: "Island Mode",
-            description: "Communication Status",
+            description: "Island Mode",
```

Comment:
Is this description correct? Island mode in the Tower Mode?

Reply:
Yes it is. It is comming from parent description as the other points but I could validate with Ivan

Reply:
KK